docs/source/user/performance-tips.rst

```diff
@@ -172,6 +172,19 @@ Execution times are as follows, ``fastmath`` again improves performance.
 | ``do_sum_parallel_fast``|      5.37 ms    |
 +-------------------------+-----------------+
 
+In some cases you may wish to opt-in to only a subset of possible fast-math
+optimizations. This can be done by supplying a set of `LLVM fast-math flags
+<https://llvm.org/docs/LangRef.html#fast-math-flags>`_ to ``fastmath``.::
+
+    def add_assoc(x, y):
+        return (x - y) + y
+
+    print(nb.njit(fastmath=False)(add_assoc)(0, np.inf)) # nan
```

Comment:
Documentation convention assumes `njit` is imported from `numba` directly so the `nb.` qualifier is not needed.